.github/workflows/kubernetes.yml

```diff
@@ -48,33 +52,33 @@ jobs:
           curl -sL -o /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.24.1/bin/linux/amd64/kubectl
           chmod +x /tmp/kubectl
           mv /tmp/kubectl /usr/local/bin/
-      - name: Create cluster
+      - name: Build and Redeploy
```

Comment:
This is kind of misleading without the context of the build and redeploy script. 'Redeploy where'?